deploy/cloudformation/godev.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	prodTemplatePath = "elastic-agent-ec2.yml"
+	devTemplatePath  = "elastic-agent-ec2-dev.yml"
+)
+
+func editArtifactURL(content string) string {
+	prodURL := "https://artifacts.elastic.co/downloads/beats/elastic-agent/"
+
+	// TODO: Dynamically get the latest snapshot URL
+	devURL := "https://snapshots.elastic.co/8.8.0-3f572553/downloads/beats/elastic-agent/"
+	return strings.ReplaceAll(content, prodURL, devURL)
+}
+
+func allowSSHFromAnywhere(content string) string {
```

Comment:
there is no way to do that in a more idiomatic way?
I would expect to have something like (instead of replacing strings)
```go
stack := Cloudformation{}
stack.Resource["ElasticAgentSecurityGroup"].Properties.GroupDescription = "Allow SSH from anywhere"
stack.Resource["ElasticAgentSecurityGroup"].Properties.SecurityGroupIngress = ...
```